packages/server/package.json

```diff
@@ -95,6 +95,10 @@
       "^.+\\.(t|j)s$": "ts-jest"
     },
     "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "moduleDirectories": [
+      "node_modules",
+      "src"
```

Comment:
I *think* this might fix the issues with the auto imports, so stuff like
```
import {UserModel} from 'profile/user.entity'
```
is all good now (don't have to do `../../`). Haven't tested it in prod tho.